src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -93,7 +93,7 @@ function WorkspaceMembersPage(props) {
      */
     const validateSelection = useCallback(() => {
         const newErrors = {};
-        const ownerAccountID = _.first(PersonalDetailsUtils.getAccountIDsByLogins([props.policy.owner]));
+        const ownerAccountID = _.first(PersonalDetailsUtils.getAccountIDsByLogins([props.policy.owner || '']));
```

Comment:
@Pujan92 i think we should fix the issue inside `getAccountIDsByLogins()` to handle undefined values.

Reply:
I am not very sure but agreed on `To avoid this issue at other places, definitely we can handle it within getAccountIDsByLogins`.

Reply:
sorry, i don't understand. Why can't we handle undefined values in `getAccountIDsByLogins()`

Reply:
can't we return early instead of pushing empty values?

Reply:
Ohh, you mean to say why to generate new accountID of the policy owner as the policy itself is not available for the user. 
https://github.com/Expensify/App/blob/0bdf18378356d6f4d5dd0aada5182b3c1648fa4e/src/libs/PersonalDetailsUtils.js#L66
`if (!currentDetail && !_.isUndefined(login)) {`
Generate new id only when the login is not undefined, does it makes sense?

Reply:
@rushatgabhane don't you think we should restrict passing undefined from the parent only and why to add check in the `getAccountIDsByLogins `?
I prefer here to update like
```
const ownerAccountID = props.policy.owner ?_.first(PersonalDetailsUtils.getAccountIDsByLogins([props.policy.owner])) : undefined;
```

https://github.com/Expensify/App/blob/2de126fc792f40b4c96d91437cd3281af8431f82/src/pages/workspace/WorkspaceMembersPage.js#L96